app/Http/Controllers/PayrollRequestController.php

```
@@ -65,6 +69,14 @@ public function createPayrollRequest(CreatePayrollRequest $request)
                 'attachment_url' => null,
             ]);
             $paymentRequest->details()->createMany($details->toArray());
+            $transactionFlowData = TransactionFlowService::getTransactionFlow(PaymentRequestType::PAYROLL->value, $paymentRequest->id);
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Services\TransactionFlowService' in method 'createPayrollRequest'. (Clean Code Rules)

(StaticAccess)